etc/profile-m-z/npm.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for npm
+# Description: The Node.js Package Manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include npm.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.npm
+noblacklist ${HOME}/.npmrc
+
+noblacklist ${PATH}/bash
+noblacklist ${PATH}/dash
+noblacklist ${PATH}/sh
+
+ignore noexec ${HOME}
+
+include disable-common.inc
+include disable-exec.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+include whitelist-common.inc
```

Comment:
wc is too much as it will not allow the most paths in your home. Maybe
```
# If you want whitelisting, change the line below to your npm-project-dir and uncomment the lines below.
#include ${HOME}/Projects
#include whitelist-common.inc
```

Reply:
Should `#include ${HOME}/Projects` instead be a `whitelist` command?

Reply:
Uhh, yes you're :100:% right.

Reply:
OK, done!